include/Arcade/Core/IGameModule.hpp

```diff
@@ -47,6 +48,12 @@ namespace Arcade {
                  *
                  */
                 virtual void changeGame() = 0;
```

Comment:
Add doc to say that it is forbiden to call this in systems
and to use event instead

Reply:
done in #77